pkg/server/storage_api/ranges_test.go

```diff
@@ -133,21 +133,58 @@ func TestRangeResponse(t *testing.T) {
 	defer log.Scope(t).Close(t)
 	defer kvserver.EnableLeaseHistoryForTesting(100)()
 	srv := serverutils.StartServerOnly(t, base.TestServerArgs{
-		DefaultTestTenant: base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(110018),
+		DefaultTestTenant: base.TestTenantProbabilistic,
 	})
 	defer srv.Stopper().Stop(context.Background())
 	ts := srv.ApplicationLayer()
 
+	t.Logf("value of StartedDefaultTestTenant %v", srv.StartedDefaultTestTenant())
+
 	// Perform a scan to ensure that all the raft groups are initialized.
 	if _, err := srv.SystemLayer().DB().Scan(context.Background(), keys.LocalMax, roachpb.KeyMax, 0); err != nil {
 		t.Fatal(err)
 	}
 
-	// TODO(#110018): grant a special capability to the secondary tenant
-	// before the endpoint can be accessed.
+	var isSystemTenant bool = !srv.StartedDefaultTestTenant()
+	t.Run("access range endpoint within current tenant's scope", func(t *testing.T) {
+		// Test case 1, looking for a range id which lies within the current tenant's (system or secondary tenant, chosen probabilistically) range
+		var currentTenantRangeId int64 // could be a system tenant or a secondary application tenant
+		t.Logf("Is system tenant enabled : %v", isSystemTenant)
```

Comment:
I think we can remove all these log statements (I'm not very convinced if they are adding value here).